action.yml

```diff
@@ -21,6 +21,12 @@ inputs:
     default: 'false'
 runs:
   using: "composite"
+  env:
+    input_cosign_release: ${{ inputs.cosign-release }}
+    input_install_dir: ${{ inputs.install_dir }}
+    input_use_sudo: ${{ inputs.use_sudo }}
+    runner_arch: ${{ runner.arch }}
+    runner_os: ${{ runner.os }}
```

Comment:
these should be set on the steps themselves, I dont think you can set them at this level